docs/technical_reference/unit_models/bipolar_electrodialysis_0D.rst

```diff
@@ -0,0 +1,340 @@
+* Constant pressure and temperature through each channel.
+* No boundary layer, electric double layer or diffusion layer, has been considered.
+
+Control Volumes
+---------------
+
+This model has two control volumes for the acidic and basic channels.
+
+* **acidic** channel
+* **basic** channel
+
+Ports
+-----
+
+On the two control volumes, this model provides four ports (Pyomo notation in parenthesis):
+
+* inlet_acidic (inlet)
+* outlet_acidic (outlet)
+* inlet_basic (inlet)
+* outlet_basic (outlet)
+
+Sets
+----
+This model can simulate the water splitting and the transport of multiple species. All solution components
+( H\ :sub:`2`\ O, neutral solutes, and ions, including Proton and Hydroxide ion) form a Pyomo set in the model.
+For a clear model demonstration, this document uses a NaCl water solution along with the products of water splitting, H\ :sup:`+` and OH\ :sup:`-`, hereafter.
+
+This model can mathematically take a multi-component (i.e., one salt molecule to be treated) as an input; nevertheless
+a multi-component solution creates unknown or difficult-to-specify parameters, e.g., the electrical transport numbers through membranes,
+the multi-ion diffusivity, etc., and physical relationships, which may result in ill-posed or ill-conditioned problems challenging the models'
+numerical solutions.  While we continuously work on advancing our models to absorb new principles revealed by progressing
+research, we advise the users be very **cautious** with simulating multi-component systems by this programmed model for aspects stated above.
+
+.. csv-table:: **Table 1.** List of Set
+   :header: "Description", "Symbol", "Indices"
+
+
+   "Time", ":math:`t`", "[t] ([0])\ :sup:`1`"
+   "Phase", ":math:`p`", "['Liq']"
+   "Component", ":math:`j`", "['H\ :sub:`2` \O', 'Na\ :sup:`+`', 'Cl\ :sup:`-`', 'H\ :sup:`+`', 'OH\ :sup:`-`']"
+   "Ion", ":math:`j`", "['Na\ :sup:`+`', 'Cl\ :sup:`-`', 'H\ :sup:`+`', 'OH\ :sup:`-`'] \  :sup:`2`"
+   "Membrane", "n/a", "['bpem']"
+
+**Notes**
+ :sup:`1` The time set index is set as [0] in this steady-state model and is reserved majorly for the future extension
+ to a dynamic model.
+
+ :sup:`2` "Ion" is a subset of "Component" and uses the same symbol j.
+
+
+Degrees of Freedom
+------------------
+The bipolar membrane model has multiple degrees of freedom, among which temperature, pressure, and component molar flow
+rate are state variables that are fixed as initial conditions. The rest are parameters that should be provided in order
+to fully solve the model. The exact degrees of freedom depend on the mode of operation. For the simplest case where no water
+splitting occurs and the bipolar membrane acts like a simple electrodialysis membrane these are:
+
+.. csv-table:: **Table 2.** List of Degree of Freedom (DOF)
+   :header: "Description", "Symbol", "Variable Name", "Index", "Units", "DOF Number \ :sup:`1`"
+
+   "Temperature, inlet_acidic", ":math:`T^acidic`", "temperature", "None", ":math:`K`", 1
```

Comment:
Please check the rst formatting in the printout throughout.  For instance, it should probably be `T^{acidic}`, etc.